### PR TITLE
Use a single term instead of all terms used to describe path and URL helpers together

### LINKED ISSUE #36233: docs fixup on named route helpers
4 different phrases are used to describe exactly the same thing. There should be some consistency in there. 
"named route helpers", "named helpers", "route helpers", "url and path helpers".

Pick a name and use it.

### PR DESCRIPTION
**Pull request by sharang-d**
Fixes #36233.

I agree with the sentiment behind the original issue: We need a single term to say "URL and Route helpers" and this is my attempt at getting there.

I'm open to suggestions for a better term if anyone has any.

### CODE REVIEW

**Review by rafaelfranca — CHANGES_REQUESTED** [review]
"named route helpers" is the correct name for this

**Comment by sharang-d** [comment]
Thanks, I'll use that term.

**Comment by sharang-d** [comment]
I've pushed the changes.

File: guides/source/routing.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ end
 
 This will create a number of routes for each of the `articles` and `comments` controller. For `Admin::ArticlesController`, Rails will create:
 
-| HTTP Verb | Path                     | Controller#Action      | Named Helper              |
+| HTTP Verb | Path                     | Controller#Action      | Route Helper                 |
```
Comment (rafaelfranca) [inline guides/source/routing.md]: should not it be `Named Route Helper`?

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

File: guides/source/routing.md (hunk 1 above)
Comment (sharang-d) [reply guides/source/routing.md]: Yes 😢 
Fixed now.

**Pull request merged**